css/main.css

```diff
@@ -1,3 +1,373 @@
+}
+
+.the-nav {
+  margin-right: 10%;
+  margin-left: 22%;
+}
+
+.blue-square {
+  background-color: blue;
+  min-width: 20px;
+  height: 20px;
+  margin-left: 5%;
+  margin-top: 15px;
+}
+
+.nav-list {
+  display: flex;
+  width: 100%;
+  flex-direction: row;
+  padding: 2%;
+  justify-content: space-between;
+  list-style-type: none;
+  font-size: 1.2rem;
+}
+
+.burger-menu {
+  display: none;
+}
+
+a:link {
+  text-decoration: none;
+  color: black;
+}
+
+a:visited {
+  color: lightblue;
+}
+
+a:hover {
+  color: blue;
+  cursor: pointer;
+}
+
+a:active {
+  text-decoration: underline;
+  color: darkblue;
+}
+
+main {
+  margin-top: 200px;
+}
+
+.first-row {
+  height: 150px;
+}
+
+.gray-div {
+  background: rgb(216, 212, 190);
+}
+
+.light-gray-div {
+  background: rgb(240, 237, 224);
+}
+
+.photo-div {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.my-photo {
+  width: 300px;
+  border-radius: 50%;
+}
+
+.small-seperator {
+  width: 80px;
+  border: 2px solid darkblue;
+}
+
+.small-photos {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  padding: 5%;
+  padding-left: 20%;
+  padding-right: 20%;
+  background-color: white;
+  width: 100%;
+  margin-bottom: 0px;
+  margin-top: 45px;
+}
+
+.small-photo {
+  width: 25px;
+}
+
+.Intro-div {
+  display: flex;
+  flex-direction: column;
+  align-content: flex-start;
+}
+
+.welcome-article {
+  margin-left: 10px;
+  font-size: 6rem;
+  font-weight: 700;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+
+.who-me {
+  margin-left: 10px;
+  font-size: 1.5rem;
+  font-weight: 300;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+
+.div-buttons {
+  display: flex;
+  justify-content: flex-start;
+  margin-left: 3%;
+  margin-top: 45px;
+}
+
+.res-btn {
+  background-color: rgb(58, 76, 240);
+  color: white;
+  font-weight: 300;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  width: 150px;
+  border-radius: 15px;
+}
+
+.proj-btn {
+  background-color: white;
+  color: rgb(58, 76, 240);
+  font-weight: 300;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  width: 150px;
+  border-radius: 15px;
+  margin-left: 20px;
+}
+
+.res-btn:hover {
+  background-color: white;
+  color: rgb(58, 76, 240);
+}
+
+.proj-btn:hover {
+  background-color: rgb(58, 76, 240);
+  color: white;
+}
+
+.text-article {
+  font-weight: 200;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  width: 50%;
+  margin-left: 20px;
+  margin-top: 60px;
+}
+
+.footer-big-row {
+  display: flex;
+  font-size: 0.9rem;
+  margin-top: 3%;
+}
+
+.smaler-photos {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  padding-left: 30%;
+  padding-right: 30%;
+  background-color: white;
+  width: 100%;
+}
+
+.last-footer-col {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.smaler-photo {
+  width: 20px;
+}
+
+.footer-small-row {
+  display: none;
+}
+
+@media (max-width: 1140px) {
```

Comment:
We want to be coding in a "mobile-first" way: this means you should write all your code to look good on mobile by default, and _then_ you add media queries (using **min**-width) to adapt it for desktop.